compose/lutris.yml

```diff
@@ -0,0 +1,48 @@
+#########################
+# lutris.yml
+#########################
+#
+# This container runs Lutris
+
+services:
+  ####################
+  lutris:
+    depends_on:
+      - sunshine
+    image: ghcr.io/games-on-whales/lutris:edge
+    build:
```

Comment:
It may be a good idea to comment out the `build:` section in the checked-in version. Historically we've done that to help prevent confusion around whether a pre-built image was being used or not; it probably isn't necessary anymore, but most of our other files still do it.

Reply:
Makes sense. This is my first PR on any github project :) - so I'll need a hand. Do I do this in my fork and then create a new PR?